README.md

```diff
@@ -7,12 +7,12 @@
 |                 | Project Info                                                     |
 | --------------- | ---------------------------------------------------------------- |
 | License:        | Apache License, Version 2.0                                      |
-| Build:          | Gradle                                                           |
+| Build:          | NPM                                                              |
```

Comment:
@paolobueno @wtrocki Don't know what to put here but I assumed "Gradle" was wrong

Reply:
Copypaste fail for me!
I guess npm is as descriptive as we can get :+1: